src/DGtal/dec/doc/packageDEC.dox

```diff
@@ -59,8 +59,9 @@ Basic operators, such as Hodge duality operator or exterior derivative, can be c
   - \ref moduleHeatLaplaceOperator (Thomas Caissard)
   - \ref moduleGenericAT (Jacques-Olivier Lachaud, Marion Foare, David Coeurjolly, Pierre Gueth)
 
-- \ref modulePolygonalCalculus (David Coeurjolly, Jacques-Olivier Lachaud)
+- \ref modulePolygonalCalculus (David Coeurjolly, Jacques-Olivier Lachaud, Baptiste Genest)
 - \ref moduleGeodesicsInHeat (David Coeurjolly, Jacques-Olivier Lachaud)
+- \ref moduleVectorsInHeat (Baptiste Genest)
```

Comment:
please add me to the module page list (not in the module page but at least here for keeping track of the future edits)